packages/pn-personagiuridica-webapp/src/components/Deleghe/ConfirmationModal.tsx

```diff
@@ -25,74 +22,40 @@ export default function ConfirmationModal({
   onConfirmLabel = 'Riprova',
   onClose,
   onCloseLabel = 'Annulla',
-  minHeight = '4em',
 }: Props) {
-  const theme = useTheme();
-  const fullScreen = useMediaQuery(theme.breakpoints.down('xs'));
-  const isMobile = useIsMobile();
-
   return (
-    <Dialog
-      fullScreen={fullScreen}
+    <PnDialog
```

Comment:
Checks the comments left on CodeModal about distance between Title and Content, and Content and Actions